ddp-workspace/projects/ddp-dsm-ui/src/app/phi-manifest/phi-manifest.component.html

```diff
@@ -0,0 +1,48 @@
+<h1> PHI Manifest Download </h1>
+
+<div *ngIf="errorMessage != null">
+  <h3 class="Color--warn Line--Break">{{errorMessage}}</h3>
+  <br/>
+</div>
+<div *ngIf="additionalMessage != null">
+  <h4 class="Color--primary Line--Break">{{additionalMessage}}</h4>
+  <br/>
+</div>
+
+<span [hidden]="!loading" class="Width--100">
+    <div align="center" class="Width--100 Height--100">
+      <i class="fas fa-spinner fa-spin fa-5x Color--primary"></i>
+    </div>
+  </span>
+
+<div *ngIf="allowedToSeeInformation">
```

Comment:
It's been a while since I've looked at the corresponding backend routes--just wanted to triple check that we are doing server-side auth checks here as well so there's no way to get the manifest by messing with client-side state.

Reply:
the `allowedToSeeInformation` gets its value from the backend call in the checkRight, more specifically here (putting code here to make sure I am addressing the right concern)
```private checkRight(): void {
        this.allowedToSeeInformation = false;
        this.dsmService.getRealmsAllowed(Statics.MEDICALRECORD).subscribe({
            next: data => {
                jsonData = data;
                jsonData.forEach((val) => {
                    if (this.realm === val) {
                        this.allowedToSeeInformation = true;
                    }
                });
                if (!this.allowedToSeeInformation) {
                    this.additionalMessage = 'You are not allowed to see information of the selected study at that category';
                }
            },
            error: () => null
        });
    }```